### PR TITLE
Nerves tutorial

### PR DESCRIPTION
**Pull request by t0mhaines**
This is the first part of the tutorial. This should take you up to a working blinking light, but without networking or a GUI.

I didn't have an LED to test with so let me know if it doesn't work as expected.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,310 @@
...
+# Nerves LED Blinking tutorial
+
+A complete tutorial of how to deploy a Nerves application on a Raspberry Pi
+</div>
+
+<br></br>
+
+## Why?
+
+Nerves is a simple to use IoT framework thats rock-solid thanks to it running on the BEAM (Erlang virtual machine). However, in order to start building
```
Comment (nelsonic) [inline README.md]: We should expand the initialisation "IoT" the first time it's used for clarity.
(_Even though the crowd reading this doc will probably be aware of it, it pays to be clear..._)

File: README.md [hunk 2]
```diff
@@ -49,45 +53,114 @@ These instructions will demonstrate how to get started from scratch.
...
+   we use the Pi Zero because it's the _cheapest_ one.
+
+   > If you want to use the _internal_ LED on your Raspberry Pi,
+   see the example on the Nerve's platform `Blinky` project: 
+   https://github.com/nerves-project/nerves_examples/tree/main/blinky
+
+   * Raspberry Pi - any version will work.
+   * 1 LED - Something like one of these form Pi Hut: 
+   https://thepihut.com/products/ultimate-5mm-led-kit
+   * 1 330 ohm resistor - without the resistor your LED will burn out!
```
Comment (t0mhaines) [inline README.md:136]: The Raspberry Pi could burn out, the resistor is very important!

Comment (t0mhaines) [reply README.md:136]: Just seen you've clarified this below :)

**Review by nelsonic — APPROVED** [review]
@th0mas this is a great start for the Nerves tutorial! 🎉 
Happy to merge this as I have tested it and it's working on my Pi Zero:
blinking LED PI Zero


I've created a new/separate issue for doing Ethernet/Wifi/USB deployment 
(not having to insert/remove the SD Card) #3 
We will need to add this ahead of deploying our home security system. 🚪

**Pull request merged**